content/posts/interview_template_name_date.md

```diff
@@ -0,0 +1,26 @@
+title: Intervju med medlem: FÖRNAMN EFTERNAMN
+date: YYYY-MM-DD
+modified: YYYY-MM-DD
+author:
+category: Interviews
+tags: interview, climate, ifk
+data:
+status: published
```

Comment:
```suggestion
status: unpublished
```

When `unpublished` the file can live in the `content/posts` folder but isn't rendered on the webpage. I guess we don't want to render the template, but it could be good to keep it around here.